packages/docusaurus-plugin-content-blog/src/__tests__/blogUtils.test.ts

```diff
@@ -71,6 +73,7 @@ describe('paginateBlogPosts', () => {
         blogTitle: 'Blog Title',
         blogDescription: 'Blog Description',
         postsPerPageOption: 10,
+        pageBasePath: 'page',
```

Comment:
That's also worth it to add a new test here with a custom `pageBasePath` (that is not "page" I mean). 

This would ensure that we actually use the method param to generate the paginated routes.

If you are not familiar with Jest snapshot testing, you can run tests with:

`yarn test packages/docusaurus-plugin-content-blog/src/__tests__/blogUtils.test.ts --watch`

Then press `u` to update snapshots. I review snapshot changes (similar to assertions), so they must contain the result we expect.

Reply:
I changed that path to `a-page`.

Unfortunately I did not get very far with running that command, it resulted in:

```
...
Test Suites: 1 passed, 1 total
Tests:       16 passed, 16 total
Snapshots:   5 passed, 5 total
Time:        0.808 s, estimated 1 s
Ran all test suites matching /packages\/docusaurus-plugin-content-blog\/src\/__tests__\/blogUtils.test.ts/i.

Active Filters: filename /packages/docusaurus-plugin-content-blog/src/__tests__/blogUtils.test.ts/
 › Press c to clear filters.

Watch Usage
 › Press a to run all tests.
 › Press f to run only failed tests.
 › Press o to only run tests related to changed files.
 › Press p to filter by a filename regex pattern.
 › Press t to filter by a test name regex pattern.
 › Press q to quit watch mode.
 › Press Enter to trigger a test run.
```

No `u` to choose from. :-(

Pressing Enter resulted in:

```
Ran all test suites matching /packages\/docusaurus-plugin-content-blog\/src\/__tests__\/blogUtils.test.ts/i.

Watch Usage: Press w to show more.
```

Reply:
For some reason I can't push to your remote branch.

Do you have this checkbox checked on the current PR?

![CleanShot 2024-02-09 at 19 53 08](https://github.com/facebook/docusaurus/assets/749374/4d150f70-9e64-4e90-a549-336d30cfc45c)

Reply:
Unfortunately not :-(

![Screenshot 2024-02-09 at 21 03 04](https://github.com/facebook/docusaurus/assets/362732/65bad693-d46c-4d32-8d8b-3cb07e46ff3d)

Reply:
If you pass me the changes in a format that I can understand, I can apply them.

Reply:
weird, maybe it's because the PR is from an organization instead of on your own account

You can cherry-pick this change: https://github.com/facebook/docusaurus/commit/f9b9396a07d955172f2d462be46074b03305def5

Reply:
> You can cherry-pick ...

Done that, thank you.